public/src/game objects/graphics/pacman arc chomp chomp.js

```diff
@@ -1,42 +1,48 @@
-var config = {
-    width: 800,
-    height: 600,
-    backgroundColor: '#010166',
-    type: Phaser.AUTO,
-    parent: 'phaser-example',
-    scene: {
-        create: create
+class Example extends Phaser.Scene {
+    constructor () {
+        super();
     }
-};
 
-var game = new Phaser.Game(config);
+    preload () {
```

Comment:
If the method has nothing in it, it doesn't need including. Better to remove it than leave it empty.